ggplot_stat_and_theme.Rmd

```diff
@@ -0,0 +1,428 @@
+# Ggplot stats and theme
+
+Keli Wang and Xinyu He
+
+```{r, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, options(warn=-1))
+
+library(ggplot2)
+library(tidyverse)
+library(showtext)
+library(ggtext)
+library(ggrepel)
+library(tibble)
+library(extrafont)
+library(readr)
+```
+
+
+## 1. Explanation and Motivation

```

Comment:
Can you remove the 1. headers (and similar below)?  The double hashtags will create the sections for you